src/componentsCommon/chackraInputFilterWithClear/chackraInputHeaderFilterWithClear.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Input, InputGroup, InputRightElement } from "@chakra-ui/react";
+import { useState } from "react";
+import { TiTimes } from "react-icons/ti";
+import { InputHeaderWithClearProps } from "./chackraInputFilterWithClear";
+
+export const ChackraInputHeaderFilterWithClear: React.FC<
```

Comment:
are you sure these components (which are nice) do play well with FinalForm? I think they should expose more parameters.

Reply:
this are for filters only, in final form config table vanilla inputs are being used iirc

Reply:
naming is bad i know